curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/match-characters-that-occur-zero-or-more-times.english.md

```diff
@@ -57,6 +57,8 @@ let result = chewieQuote.match(chewieRegex);
 <section id='solution'>
 
 ```js
-// solution required
+  let chewieQuote = "Aaaaaaaaaaaaaaaarrrgh!";
+  let chewieRegex = /A[a]*/;
```

Comment:
This is the correct solution, but the brackets are useless here.

Reply:
That is true.  It could just be `/Aa*/`.  @luane91 Please update your solution.